README.md

```diff
@@ -142,6 +146,15 @@ add_action('simply_static_deploy_error', function (\WP_Error $error) {
 });
 ```
 
+#### Completed static deploy job
+This will be triggered after al deploy tasks are finished. The first and only argument you will get
+in the callback function is the Simply Static options instance.
```

Comment:
```suggestion
#### Completed static deploy job

This will be triggered after all deploy tasks are finished. The first and only argument you will get
in the callback function is the Simply Static options instance.
```